src/actions/pause/paused.js

```diff
@@ -69,6 +71,7 @@ export function paused(pauseInfo: Pause) {
       dispatch(removeBreakpoint(hiddenBreakpointLocation));
     }
 
+    await dispatch(setSymbols());
```

Comment:
i think one concern here is that we could already be getting the symbols for this source and we wouldn't want to try to get it a second time.

This is why we have the logic in loadSourceText to check for this... and some unit tests to also check for it.

Reply:
Okay i understood the logic in `loadSourceText`.  I need to change `setSymbols` function in ast.js and perform fetch as an `loadSourceText` correct me if i am wrong. @jasonLaster

Reply:
That's right @nagpalm7. We'll need something like this

```js
    // Fetch the source text only once.
    if (requests.has(id)) {
      return requests.get(id);
    }


...

    // signal that the action is finished
    deferred.resolve();
    requests.delete(id);
```